docs/database/schema-alternative.sql

```diff
@@ -0,0 +1,101 @@
+--
+-- Copyright (C) 2016 Red Hat, Inc.
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--         http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+--
+
+CREATE EXTENSION hstore;
+
+-- ==== Instances ====================================================================
+
+-- An action instance is the single entry point in this sub domain. It is referenced
+-- by an integration and has all configuration needed for creating an instance of
+-- a fully configured Camel connector
+CREATE TABLE action_instance
```

Comment:
While I agree that this approach seems to make more sense and I also prefer to have a single source of truth (the DB in this case), I still have a problem with how this can work once we have expert developers hacking on git repos. Do we defer that for now and figure it out when it becomes something we need to worry about? I'm happy to do that so we can get this moving.